src/main/js/ephox/imagetools/transformations/ImageTools.js

```diff
@@ -80,13 +82,13 @@ define("ephox/imagetools/transformations/ImageTools", [
 
   function resize(blob, w, h) {
     return Conversions.blobToImage(blob).then(function(image) {
-      var canvas = Canvas.create(w, h),
-        context = Canvas.get2dContext(canvas);
-
-      context.drawImage(image, 0, 0, w, h);
-      revokeImageUrl(image);
-
-      return Conversions.canvasToBlob(canvas, blob.type);
+      return ImageResizerWebgl.scale(image, w, h)
```

Comment:
Maybe have an options object as another argument, where we pass additional params? We could even pass in destination width and height as key-value params in this options object. Or FP approach explicitly requires to have each argument passed in separately?